ff/serving/provider/snowflake.go

```diff
@@ -0,0 +1,631 @@
+package provider
+
+import (
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"fmt"
+	sf "github.com/snowflakedb/gosnowflake"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// snowflakeColumnType is used to specify the column type of a resource value.
+type snowflakeColumnType string
+
+const (
+	SFInt    snowflakeColumnType = "integer"
```

Comment:
We really should have generic type schemas and they should map into these things. That'd also allow us to make this consts be private

Reply:
I know you're working on it, and you should merge this, just want to make sure it doesn't get forgotten

Reply:
Yeah I was going to rework a lot of the Type stuff when doing the generic tables/transformations. Will get it in the next PR